src/webgpu/util/pretty_diff_tables.ts

```diff
@@ -1,5 +1,27 @@
 import { range } from '../../common/util/util.js';
 
+/**
+ * @returns a function that converts numerics to strings, depending on if they
+ * should be treated as integers or not.
+ */
+export function numericToStringBuilder(is_integer: boolean): (n: number | bigint) => string {
+  if (is_integer) {
+    return (val: number | bigint): string => {
+      if (typeof val === 'number') {
+        return val.toFixed();
```

Comment:
I'm curious: Why do we use `toFixed()` if we want to print an integer?

Reply:
I didn't write this code originally, but quickly looking at .toFixed(), it has a arg `fractionDigits` which defaults to 0, which means don't print any digits after the decimal point (and the decimal point). So this would be doing the same thing as trunc'ing the value.

I am not sure why this is needed, since the value is supposed to be an integer, and an integer would be displayed the same way using .toString().
I assume there is some way that an non-int might get in here? :man_shrugging:

Reply:
Okidoki.